src/components/app.tsx

```diff
@@ -157,10 +160,10 @@ const App = () => {
 
     useEffect(() => {
         initialize()
-            .then((userManager) => {
-                setUserManager({ instance: userManager, error: null });
+            .then((userManager: UserManager | undefined) => {
+                setUserManager({ instance: userManager || null, error: null });
```

Comment:
that is intentional from me because an empty object is the same as null for authentication system, and it permit to use `?.` operator in the rest of code

@TheMaskedTurtle you have work on the auth part recently, I am wrong in my assumptions?

Reply:
Actually I think it either throws or the UserManager will be defined. But yes more generally handle empty object case is not a good practice, if we didn't manage to create our object we should return null or undefined not empty object.